karma.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
# RE Karma costs...

See [Developer Productivity Economics](https://twitter.com/_ericelliott/status/572924586775298048)

A context switch is similar to an interruption.